wolox-training/mr-rails
# Adding Twitter Bootstrap

# Summary
* Adding Twitter Bootstrap for giving style to the blog

# Test plan
1. Go to the home page
2. See the Bootrstap navbar and the styles in the buttons and tables

# Trello card
https://trello.com/c/jhjwo6B8/11-instalar-y-configurar-twitter-bootstrap
